docs/diagnostics/README.md

```diff
@@ -7,7 +7,7 @@ to specific code patterns identified through static analysis.
 
 ## Experimental APIs
 
-Range: OTEL1XXX
+Range: OTEL1000 - OTEL1999
```

Comment:
This change seems to make things more clear, but What's Policheck? Was OTEL1XXX caught by some lint check or something?

Reply:
> What's Policheck?

It is a tool used inside Microsoft to check potential offensive/sensitive words in codebase.

> Was OTEL1XXX caught by some lint check or something?

It was discovered by a Microsoft product which takes OpenTelemetry .NET source code. The story is similar to https://bugs.chromium.org/p/chromium/issues/detail?id=981129.

Reply:
So, you're saying OpenTelemetry .NET is now rated PG rather than XXX? 😆